group1/comm/static/comm_tool.js

```diff
@@ -322,6 +324,8 @@ function switch_room(target_room){
   // reset the badge count for the target room
   $('div#room-list a').filter( function(){ return $(this).attr('id') === target_room } ).children().filter('.badge').text('');
 
+  document.getElementById("bottom").scrollIntoView();
```

Comment:
This looks pretty good. We would typically want to use more descriptive names for element IDs other than "bottom" though. Just something to keep in mind for the future.